client/hotrod-client/src/main/java/org/infinispan/client/hotrod/RemoteCacheManager.java

```diff
@@ -490,6 +496,15 @@ public ChannelFactory getChannelFactory() {
       return channelFactory;
    }
 
+   /**
+    * Returns the {@link XAResource} which can be used to do transactional recovery.
```

Comment:
and what is the best way to achieve recovery? Enlist this Xa in a transaction :) 
The TM can keep a reference to it and invoke the recovery methods. 
In addition, if the user wants to do some "manual" recovery, it can get the Xa and invoke itself, but I'm assuming this case won't be used.

Reply:
@pruivo I'm not saying it cannot be used for recovery, I was just complaining that only the javadoc says that and doesn't mention that it can be used to enlist a cache in a transaction, while the JIRA/PR/commit message say it can be used to enlist the cache in a tx but never mention recovery.

I'm still not sure how recovery is supposed with different TMs (e.g. Narayana vs. Atomikos) so I would have avoided any reference to recovery :)